website/docs/language/settings/backends/configuration.mdx

```diff
@@ -42,6 +42,7 @@ There are some important limitations on backend configuration:
 
 - A configuration can only provide one backend block.
 - A backend block cannot refer to named values (like input variables, locals, or data source attributes).
+- You cannot [reference](https://www.terraform.io/language/expressions/references#references-to-resource-attributes) values declared within backend blocks elsewhere in the configuration.
```

Comment:
@sebhcp Thank you for this contribution! We recently migrated our documentation to the new developer portal site, so I wanted to update this in-text link accordingly. Also, I want to make sure the link text is as descriptive as possible so users know where it will take them.

Let me know what you think - thank you!

```suggestion
- You cannot reference values declared within backend blocks elsewhere in the configuration. Refer to [References to Resource Attributes](/terraform/language/expressions/references#references-to-resource-attributes) for more details.
```